run_build.sh

```diff
@@ -58,8 +58,9 @@ if [ "${DAOS_BRANCH}" == "release/1.2" ]; then
 else
     # Use the newer branch
     declare -a DAOS_PATCHES=(
+                             "origin/dbohninx-io500-base-daaf038"
+                             #"origin/dbohninx-io500-base-5d740e5"
```

Comment:
Any reason to still keep the older one? Maybe we can remove the next line origin/mjmac/io500-202104? too?